### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by spences10**
Changes still need review before merging

### CODE REVIEW

**Comment by spences10** [comment]
@amandeepmittal I haven't tested the changes here my friend - I'm in the process of installing snyk to apply some patches

**Comment by spences10** [comment]
@amandeepmittal this was meant as a placeholder PR for now 🙃

File: package.json [hunk 1]
```diff
@@ -26,21 +26,21 @@
   },
   "license": " BSD 3-Clause License",
   "devDependencies": {
-    "jest-cli": "21.2.1",
-    "nodemon": "1.12.1",
+    "jest-cli": "22.0.4",
```
Comment (spences10) [inline package.json]: @amandeepmittal possible breaking change here 🙃

Comment (amandeepmittal) [reply package.json]: You want me to revert back the PR? Or is everything fine with it? I thought you meant that you have updated dependencies.

**Pull request merged**